docs/kyma/04-02-local-installation.md

```diff
@@ -19,107 +20,94 @@ Virtualization:
+  
+  2. Install the latest Kyma release on Minikube:
+     ```bash
+     kyma install 
+     ```
+     >**NOTE** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling ` kyma install --release {KYMA_RELEASE}`.
+   
+  </details>
+  <details>
+  <summary>
+  From sources
+  </summary>
+    
+  1. Open a terminal window and navigate to a space in which you want to store local Kyma sources.
+    
+  2. Clone the `Kyma` repository using HTTPS. Run:
+     
+     ```bash
+     git clone https://github.com/kyma-project/kyma.git
+     ```
+  3. Provision a Kubernetes cluster on Minikube. Run:
+     ```bash
+     kyma provision minikube
+     ```
+     >  **NOTE:** The `provision` command uses default Minikube VM driver installed for your OS. For a list of supported VM drivers see [this document](http://github.com/kyma-project/cli).
+
+  4. Install Kyma from sources. Run:
+     
+     ```bash
+     kyma install --local --src-path {YOUR_KYMA_SOURCE_PATH}
+     ```
+
+   </details>
+</div>
+
+ ## Post-installation steps
+  1. Kyma comes with a local wildcard self-signed `server.crt` certificate. Add this certificate to your OS trusted certificates to access the Console UI. 
+     
+     To download certificate, run:
+     ```bash
+     wget https://github.com/kyma-project/kyma/blob/master/installation/certs/workspace/raw/server.crt
+     ```
+     <div tabs>
       <details>
       <summary>
-      Linux
+         On MacOS, run:
       </summary>
 
+      ```bash
+      sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain server.crt
       ```
-      certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n {CERT_DISPLAYNAME} -i certs/workspace/raw/server.crt
-      ```
+      
       </details>
-    </div>
-
-  >**NOTE:** Mozilla Firefox uses its own certificate keychain. If you want to access the Console UI though Firefox, add the Kyma wildcard certificate to the certificate keychain of the browser. To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read [this](/components/application-connector#details-access-the-application-connector-on-a-local-kyma-deployment) document to learn more.
-
-7. Start the installation. Trigger the `run.sh` script to start Minikube with a Kyma-specific configuration and install the necessary components. Define the password used to log in to the Console UI using the `--password` flag. Run:
-    <div tabs>
       <details>
       <summary>
-      MacOS
+         On Linux, run:
```

Comment:
I don't really like the flow of this section the way it is now, it's kinda broken. 
The names of the tabs should be simply "MacOS / Linux". Would it work if you put two commands into each step (download and add to keychain) but separated with a `;` separator so that when the user copies the content of the code block into the terminal he'll just have to press enter to both download the cert and add it to trusted? Like this: 


```
1. Kyma comes with a local wildcard self-signed `server.crt` certificate. Download the certificate from Kyma GitHub and it to your OS trusted certificates to access the Console UI. Run: 
 <div tabs>
      <details>
      <summary>
      MacOS
      </summary>

      ```bash
       wget https://github.com/kyma-project/kyma/blob/master/installation/certs/workspace/raw/server.crt ; sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain server.crt
      ```
      
      </details>
      <details>
      <summary>
       Linux
      </summary>

      ```bash
      wget https://github.com/kyma-project/kyma/blob/master/installation/certs/workspace/raw/server.crt ; certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n {CERT_DISPLAYNAME} -i server.crt
      ```
      
      </details>
   </div>